processing/file_fixer.py

```diff
@@ -1309,7 +1309,9 @@ def upload_info(self, payload):
         upload_info_url = self.upload_ws1
         try:
             payload_json = json.dumps(payload)
-            response = requests.post(url=upload_info_url, data=payload_json)
+            response = requests.post(
+                url=upload_info_url, data=payload_json,

```

Comment:
Based on my tests, I think the kwarg should be `json=payload_json` rather than `data=payload_json`. I was just trying the request outside of the full file_fixer context, but when I used `data` with or without the header, my request failed. When I used `json`, with or without the header, it was successful. Not sure if this is something you're seeing as well.

Reply:
What python and requests versions are you using?
The code worked on the dev system (python 3.10.12 and requests 2.31.0) with `data`.

Reply:
Using `json` in the kwargs on the dev system yielded 422 error.

Reply:
I see what I did wrong. I missed the line above where you dumped the payload from a dict into a string. Tried it out with the `json.dumps` and submitting that value to `data` and it totally worked. Saw the same behavior you did. My bad!